vms/habcloud-vms

```diff
@@ -497,27 +533,35 @@ def main():
     xml.set_defaults(func=cmd_xml)
     xml.add_argument("vm_name", help='guest hostname')
 
-    dnsmasq_dhcp = subparsers.add_parser("dnsmasq-dhcp", help='dnsmasq dhcp-hosts file')
+    dnsmasq_dhcp = subparsers.add_parser("dnsmasq-dhcp",
+                                         help='dnsmasq dhcp-hosts file')
     dnsmasq_dhcp.set_defaults(func=cmd_dnsmasq_dhcp)
-    dnsmasq_dhcp.add_argument("host", help='VM host', choices=HOST_NETWORKS.keys())
+    dnsmasq_dhcp.add_argument("host", help='VM host',
+                              choices=HOST_NETWORKS.keys())
 
-    dnsmasq_hosts = subparsers.add_parser("dnsmasq-hosts", help='dnsmasq hosts file')
+    dnsmasq_hosts = subparsers.add_parser("dnsmasq-hosts",
+                                          help='dnsmasq hosts file')
     dnsmasq_hosts.set_defaults(func=cmd_dnsmasq_hosts)
 
-    create = subparsers.add_parser("create", help='create an allocate VM on this box')
+    create = subparsers.add_parser("create",
+                                   help='create an allocate VM on this box')
     create.set_defaults(func=cmd_create)
     create.add_argument("vm_name", help='guest hostname')
 
-    sync_dnsmasq = subparsers.add_parser("sync-dnsmasq", help='sync and HUP dnsmasq on this box')
+    sync_dnsmasq = subparsers.add_parser(
```

Comment:
I'm becoming less convinced by line length limits. I think this looks a bit silly. The 80 width limit is a bit...outdated. Even if you use a width 80 terminal vim line numbers typically mean len 80 lines don't fit.